src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -238,15 +240,20 @@ private void createInfoPanel() {
         TitledBorder tb = new TitledBorder(new LineBorder(Color.darkGray),
                                            I18nUtils.getString("EXAM_ADDRESS_1"));
         infoPanel_.setBorder(tb);
+       
         infoPanel_.setToolTipText(I18nUtils.getString("EXAM_DISPLAY_2"));
+        String unUsedString = "Hello World ";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "unUsedString" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)